backend/src/order.ts

```diff
@@ -89,24 +89,24 @@ export async function checkPaymentStatus(order_id: number): Promise<string> {
   [res, err] = await to(db.query('select payment_id, payment_url from nk2_orders where id = :order_id', {order_id}));
   if (err) {
     log.error('Check payment status failed', {error: err, order_id});
-    return reject(err);
+    throw err;
   }
 
   log.log('debug', 'Create payment handler', {order_id});
   [handler, err] = await to(getPaymentHandler(order_id));
   if (err) {
-    return reject(err);
+    throw err;
   }
```

Comment:
This is functionally equivalent to 
`let handler = await getPaymentHandler(order_id)`

do you really prefer this?

Reply:
I'll know, but as it seems, that I'm only one favoring for golang style error handling and I can live with both, I'll refactor them to use try/catch syntax.